### ISSUE TITLE
"Lower off" tag

### ISSUE THREAD

**Issue by skomaroff** [theCrag/website#4115]
Currently there are three tags for descent options:

- Walk off
- Scramble off
- Rap off

Another common (but occasionally contentious) option is to lower off rather than rappel.  I think it would be good to have a tag available to mark locations where lowering off is the local consensus.

**Comment by nicHoch** [comment]
Could you please try to define where exactly is the difference between  "lower off" and "rap off" and what it implies for the climber

**Comment by skomaroff** [comment]
The difference is in whether the weighted rope runs through the anchor.  
When rappelling, only the unweighted rope is pulled through the fixed hardware.
Lowering causes more wear on the anchor, and so in some regions it has historically been controversial.  Most areas seem to be moving towards lowering as the default.

From my understanding, few people differentiate in Europe.  I know first hand that many people in USA, Canada and New Zealand care about the distinction.

Just some of many discussions:

- https://outdoors.stackexchange.com/questions/27163/how-do-i-know-the-local-ettiquete-on-lowering-vs-rappeling
- https://www.rockandice.com/climbing-gear-tips/should-you-lower-or-rap-through-anchors/
- https://www.rockandice.com/climbing-gear-tips/is-it-okay-to-lower-off-rap-rings/
- https://old.reddit.com/r/climbing/comments/7tqq1h/rappel_or_lower_off_a_sport_route/
- https://old.reddit.com/r/climbing/comments/70z0nj/question_regarding_rappelling_vs_lowering/
- https://outdoors.stackexchange.com/questions/6483/lowering-off-from-anchors-vs-rappelling-after-a-climb
- https://www.mountainproject.com/forum/topic/107279145/lowering-vs-rappelling

**Comment by killakalle** [comment]
It sounds like Walk off, Scramble off, Rap off are used in the context of
multi-pitch.

Does "Lower off" even exist for multi-pitch? Or would you want to use it in
a spot climbing context?

**Comment by skomaroff** [comment]
Correct, lower off is most relevant to single pitch (sport and trad).

All the other tags are relevant to single pitch too though; particularly trad crags where permanent anchors may not exist.  

"Walk off" and "scramble off" are also relevant to bouldering.  A "down climb" tag might even be useful for boulders that have more technical descents.
